include/boot_img_hdr.h

```diff
@@ -79,6 +81,53 @@ struct boot_img_hdr
     uint32_t id[8]; /* timestamp / checksum / sha1 / etc */
 };
 
+struct boot_img_hdr_elf
+{
+    uint8_t magic[BOOT_MAGIC_SIZE]; /* .ELF (0x00 to 0x07) */
```

Comment:
No in fact the ELF magic is 8 too, but the recognized chars are ELF (\x7fELF\x01\x01\x01\x61)